battleground_viewer/src/construct_render/render/primitive_geometry.rs

```diff
@@ -0,0 +1,192 @@
+use three_d::*;
+
+use super::batching::*;
```

Comment:
This is redundant with `use super::*` and super itself doing `use batching::*`, but it's a minor thing.